src/user-guide/task-implementation/task-run-modes.rst

```diff
@@ -0,0 +1,119 @@
+.. _task-run-modes.skip:
+
+Skip Mode
+=========
+
+.. versionadded:: 8.4.0
+
+Skip mode is designed as an aid to workflow control:
+
+* It allows creation of :term:`dummy tasks<dummy task>` as part of workflow
+  design.
+* It allows skipping of tasks in a running workflow using either:
+  * ``cylc broadcast -s 'run mode = skip'`` (for when it is ready to run)
+  * ``cylc set --out skip`` (to immediately skip)
+
+.. note::
+
+   Setting :cylc:conf:`[runtime][<namespace>]run mode` in your ``flow.cylc``
+   will lead to ``cylc validate`` returning a warning::
+
+      WARNING - The following tasks are set to run in skip mode:
+        * example_task
+
+   This is designed to prevent users running a task in skip mode by mistake.
+   If you are using skip mode deliberately then this can be
+   ignored.
+
+Task Outputs
+^^^^^^^^^^^^
+
+Skip mode allows the user to specify which task outputs
+will be emitted using :cylc:conf:`[runtime][<namespace>][skip]outputs`.
+
+By default:
+
+* All required outputs will be generated.
+* ``succeeded`` will be generated even if success is :term:`optional <optional output>`.
+* If ``[skip]outputs`` is specified and does not include either ``succeeded``
+  or ``failed``, then ``succeeded`` will still be generated.
+
+The outputs submitted and started are always generated and do not
+need to be defined in outputs.
+
+Task Event Handlers
+^^^^^^^^^^^^^^^^^^^
+
+By default task event handlers are disabled by skip mode, but they
+can be enabled using
+:cylc:conf:`[runtime][<namespace>][skip]disable task event handlers`.
+
+Skip Mode Examples
+------------------
```

Comment:
This isn't the right heading level

Reply:
Is it not? Or is our theme not displaying headings right?

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Happy to bump it up a level.

Reply:
```suggestion
Skip Mode Examples
==================
```